### PR TITLE
Finish migration to track_decl.h

### PR DESCRIPTION
**Pull request by uklotzde**
Only technical refactorings to make it compile.

### CODE REVIEW

File: src/library/banshee/bansheeplaylistmodel.cpp [hunk 1]
```diff
@@ -75,71 +77,37 @@ void BansheePlaylistModel::setTableModel(int playlistId) {
...
-                 CLM_GENRE " TEXT, "
-                 CLM_GROUPING " TEXT, "
-                 CLM_TRACKNUMBER " INTEGER, "
-                 CLM_DATEADDED " INTEGER, "
-                 CLM_BPM " INTEGER, "
-                 CLM_BITRATE " INTEGER, "
-                 CLM_COMMENT " TEXT, "
-                 CLM_PLAYCOUNT" INTEGER, "
-                 CLM_COMPOSER " TEXT, "
-                 CLM_PREVIEW " TEXT)");
```
Comment (daschuer) [inline src/library/banshee/bansheeplaylistmodel.cpp:98]: Unrelated clang-format attack, please revert. 
I guess there is something wrong with your editor.

File: src/library/banshee/bansheeplaylistmodel.cpp [hunk 2]
```diff
@@ -75,71 +77,37 @@ void BansheePlaylistModel::setTableModel(int playlistId) {
...
+                ", " CLM_GENRE ", " CLM_GROUPING ", " CLM_TRACKNUMBER
+                ", " CLM_DATEADDED ", " CLM_BPM ", " CLM_BITRATE
+                ", " CLM_COMMENT ", " CLM_PLAYCOUNT ", " CLM_COMPOSER
+                ") "
+                "VALUES (:" CLM_TRACK_ID ", :" CLM_VIEW_ORDER ", :" CLM_ARTIST
+                ", :" CLM_TITLE ", :" CLM_DURATION ", :" CLM_URI ", :" CLM_ALBUM
+                ", :" CLM_ALBUM_ARTIST ", :" CLM_YEAR ", :" CLM_RATING
+                ", :" CLM_GENRE ", :" CLM_GROUPING ", :" CLM_TRACKNUMBER
+                ", :" CLM_DATEADDED ", :" CLM_BPM ", :" CLM_BITRATE
+                ", :" CLM_COMMENT ", :" CLM_PLAYCOUNT ", :" CLM_COMPOSER ") ");
```
Comment (daschuer) [inline src/library/banshee/bansheeplaylistmodel.cpp:110]: the same here, please revert.

**Review by daschuer — CHANGES_REQUESTED** [review]
I think it looks good so far all the unrelated edits are quite annoying though.

File: src/library/banshee/bansheeplaylistmodel.cpp (hunk 2 above)
Comment (uklotzde) [reply src/library/banshee/bansheeplaylistmodel.cpp:110]: The same issue as in the other PR, the pre-commit hook kicked in.

**Comment by uklotzde** [comment]
Force push: I have reverted all unrelated edits probably caused by the pre-commit hook.

**Comment by uklotzde** [comment]
@daschuer I found the cause of the reformatting: VSCode doesn't respect the "only format modifications on save" setting while resolving conflicts! It was not the hook.

Unfortunately, I have to resolve conflicts regularly to keep my pending branches up-to-date :((

**Review by Holzhaus — APPROVED** [review]
LGTM, thank you!

**Review by daschuer — APPROVED** [review]
Thank you. LGTM

**Pull request merged**